Automattic/jetpack
# Launchpad: Remove unnecessary require

Related to [#75504](https://github.com/Automattic/wp-calypso/issues/75504)

## Proposed changes:

* Removes a duplicated `require` of the Launchpad code; this code is already required in the features index: https://github.com/Automattic/jetpack/blob/trunk/projects/packages/jetpack-mu-wpcom/src/class-jetpack-mu-wpcom.php#L80
## Does this pull request change what data or activity we track or use?

No.

### Other information:


## Jetpack product discussion
https://github.com/Automattic/wp-calypso/issues/75504


## Testing instructions:






* Apply this patch to your sandbox and sandbox `public-api.wordpress.com`
* Visit `/start` and create a new site with the "Write & Publish" intent.
* The Launchpad step should still work, with no errors from the API.
* Your sandbox should not throw any PHP errors as a result of removing this require.
